os/arch/arm/src/stm32/stm32_otgfshost.c

```diff
@@ -950,7 +1026,7 @@ static int stm32_chan_wait(FAR struct stm32_usbhost_s *priv, FAR struct stm32_ch
 		 * awakened by a signal too.
 		 */
 
-		DEBUGASSERT(ret == OK || get_errno() == EINTR);
+		DEBUGASSERT(ret == OK || ret == -EINTR);
```

Comment:
sem_wait does not return errno. It returns OK or ERROR . Previous code is right.

Reply:
Done